dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/DatadogEventBridge.kt

```diff
@@ -49,4 +49,24 @@ class DatadogEventBridge {
         }
         return origins.toString()
     }
+
+    // endregion
+
+    // region internal
+
+    private fun getWebEventConsumer(): WebEventConsumer {
```

Comment:
Is there a reason why you're using this instead of a `by lazy (mode = LazyThreadSafetyMode.SYNCHRONIZED){ … }`?

Reply:
hmmm...didn't know about that trick...I am going to have a look.

Reply:
Ok I have my answer for this, I actually did try with lazy but apparently you cannot use `lazy` with a `var` and I needed a var there for my unit tests. There is also not possible to have this in the constructor as this can create problems: "the bridge was created before the sdk was initialized" and we do not want that.

Reply:
In which case can we have a situation when bridge is created before the SDK is initialized?